pom.xml

```diff
@@ -433,16 +433,6 @@ and
   <build>
     <defaultGoal>test</defaultGoal>
     <plugins>
-      <plugin>
-        <artifactId>maven-source-plugin</artifactId>
-        <executions>
-          <execution>
-            <goals>
-              <goal>jar</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
```

Comment:
This is already defined in https://github.com/jenkinsci/pom/blob/master/pom.xml#L738-L748, causing the build to fail, given it's now declared twice.